seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-7/src/test/resources/jdbc_xugu_source_and_upset_sink.conf

```diff
@@ -0,0 +1,48 @@
+#
```

Comment:
rename `jdbc_xugu_source_and_upset_sink.conf` to `jdbc_xugu_source_and_upsert_sink.conf`

Reply:
ok